services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,38 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  if(xQueueSend(thermalMgrQueueHandle, event, (TickType_t) 10) != pdPASS) return ERR_CODE_INVALID_QUEUE_MSG;
```

Comment:
Before using `thermalMgrQueueHandle`, check if `thermalMgrQueueHandle == NULL`. If it is, then return ERR_CODE_INVALID_STATE since it'd mean the firmware is in a weird state.

Reply:
Done, will push change.